include/core/boundary_conditions.h

```diff
@@ -678,6 +681,162 @@ namespace BoundaryConditions
     prm.leave_subsection();
   }
 
+  /**
+   * @brief This class manages the boundary conditions for VOF solver
+   * It introduces the boundary functions and declares the boundary conditions
+   * coherently. (TODO update)
+   * The members ... contain double used for bc calculation :
+   *  - if bc type is "..." (... condition), "..." is the
+   * double passed to the deal.ii ConstantFunction
```

Comment:
I don't understand this part?

Reply:
thank you for spotting that I forgot to update that :)